server/routerlicious/packages/lambdas-driver/src/kafka-service/partitionManager.ts

```diff
@@ -57,23 +57,34 @@ export class PartitionManager extends EventEmitter {
 				this.emit("error", error, errorData);
 			});
 
-			this.consumer.on("checkpoint_success", (partitionId, queuedMessage) => {
-				if (this.sampleMessages(100)) {
-					Lumberjack.info(`Kafka checkpoint successful`, {
+			this.consumer.on(
+				"checkpoint_success",
+				(partitionId, queuedMessage, retries, latency) => {
+					if (this.sampleMessages(100)) {
+						Lumberjack.info(`Kafka checkpoint successful`, {
+							msgOffset: queuedMessage.offset,
+							topic: queuedMessage.topic,
+							msgPartition: queuedMessage.partition,
+							retries,
+							latency,
+						});
+					}
+				},
+			);
+
+			this.consumer.on(
+				"checkpoint_error",
+				(partitionId, queuedMessage, retries, latency, ex) => {
+					Lumberjack.error(`Kafka checkpoint failed`, {
 						msgOffset: queuedMessage.offset,
 						topic: queuedMessage.topic,
 						msgPartition: queuedMessage.partition,
+						retries,
+						latency,
+						exception: ex,
```

Comment:
NIt, don't need to bake the exception in property, 3rd parameter takes exception:
![image](https://github.com/microsoft/FluidFramework/assets/3719246/348d07a1-f49b-4841-9543-f02ef37d7c41)